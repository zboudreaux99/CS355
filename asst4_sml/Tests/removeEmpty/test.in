use "remove.sml";
val actual = remove(42, []);
val expected = [];
if actual = expected then "PASSED 100/100" else "FAILED 0/100";
