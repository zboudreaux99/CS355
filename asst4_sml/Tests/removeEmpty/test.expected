sml: unable to determine architecture/operating system
